src/components/LoanCards/HoverLoanCard/InfoPanels/BorrowerStoryPanel.vue

```diff
@@ -3,31 +3,90 @@
 		<template #title>
 			Borrower story
 		</template>
-		<p>Lot's of stories and text.</p>
+		<p
+			v-if="loanStory"
+			v-html="loanStory"
+		></p>
+		<div v-else id="loading-overlay">
+			<div class="spinner-wrapper">
+				<kv-loading-spinner />
+			</div>
+		</div>
+		<router-link :to="`/lend/${loanId}`">
+			Read full story
+		</router-link>
 	</info-panel>
 </template>
 
 <script>
+import _get from 'lodash/get';
 import InfoPanel from './InfoPanel';
+import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
+import loanDescriptionQuery from '@/graphql/query/loanDescription.graphql';
 
 export default {
 	components: {
 		InfoPanel,
+		KvLoadingSpinner
 	},
+	inject: ['apollo'],
 	props: {
 		expandable: {
 			type: Boolean,
 			default: true,
 		},
 		loanId: {
-			type: String,
-			default: '',
+			type: Number,
+			default: 0,
 		},
 	},
+	data() {
+		return {
+			loanStory: ''
+		};
+	},
+	created() {
+		this.apollo.query({
+			query: loanDescriptionQuery,
+			variables: {
+				id: parseInt(this.loanId, 10)
+			}
+		}).then(({ data }) => {
+			this.loanStory = _get(data, 'lend.loan.description');
+		});
```

Comment:
Instead of this custom created function, you should take advantage of the apollo plugin so that this component will also update when the loan id changes
```
apollo: {
  query: loanDescriptionQuery,
  result({data}) {
    this.loanStory = _get(data, 'lend.loan.description');
  },
}
```
}

Reply:
Sorry @emuvente @BoulderBrains had apollo setup initially but the variables on 'this.' weren't coming through... so we moved it. I think he later found the issue was higher up and the var wasn't being passed into the component.